client.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<link href="diff-stylesheet.css" rel="stylesheet">
+	<script type="text/javascript">
+		'use strict';
+
+		function onNewContent() {
+			var httpRequest = ASMDW.httpRequest;
+			if (httpRequest.readyState === XMLHttpRequest.DONE) {
+				if (httpRequest.status === 200) {
+					document.getElementById('content').innerHTML = httpRequest.responseText;
+					if (ASMDW.requestForever) {
+						setTimeout(requestContent, 100);
+					}
+				}
+				else {
+					if (window.confirm('Failed to communicate with localhost:8000, close tab?')) {
```

Comment:
it may be nice to show a red status marker instead, and keep polling (with exponential backoff?). Then you won't get a prompt every time you diff.py a new function, and can instead continue using the same tab. (But this is fine for now ofc)